[DOCS-1383] Update billing to docs to reflect changes to tracked hours billing (#2073)
wandb/docs: support/models/articles/how-can-i-see-the-bytes-stored-bytes-tra.mdx

Author: Needs to be updated in DocEngine instead

Ensure to replace values enclosed in `<>` with your organization's name.

<Note>

> Tracked hours are unlimited for all plans and are no longer billed. While tracked hours are still displayed on the Usage page for monitoring purposes, your organization will not be charged for them.

Author: ```suggestion
Tracked hours are unlimited for all plans and are not billed. Tracked hours are displayed on the **Usage** page for monitoring purposes only.
```

Ensure to replace values enclosed in `<>` with your organization's name.

<Note>

> Tracked hours are unlimited for all plans and are no longer billed. While tracked hours are still displayed on the Usage page for monitoring purposes, your organization will not be charged for tracked hour usage or overages.

Reviewer: ```suggestion
Tracked hours are unlimited for all plans and are no longer billed. While tracked hours are still displayed on the Usage page for monitoring purposes, your organization will not be charged for them.
```
